src/cli/commands/dns.js

```diff
@@ -7,15 +7,21 @@ module.exports = {
   describe: 'Resolve DNS links',
 
   builder: {
+    recursive: {
+      type: 'boolean',
+      default: true,
+      alias: 'r',
+      desc: 'Resolve until the result is not a DNS link'
+    },
     format: {
       type: 'string'
     }
   },
 
-  handler ({ getIpfs, domain, resolve }) {
+  handler ({ getIpfs, domain, resolve, ...opts }) {
```

Comment:
Please could you use long hand for now - I don't know how good support for this feature is in browsers/Node.js and we don't transpile.

```suggestion
  handler ({ getIpfs, domain, resolve, recursive, format }) {
```